lib/widgets/profile.dart

```diff
@@ -0,0 +1,259 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+
+import '../api/model/model.dart';
+import '../model/content.dart';
+import '../model/narrow.dart';
+import 'content.dart';
+import 'message_list.dart';
+import 'page.dart';
+import 'store.dart';
+
+class _TextStyles {
+  static const profileText = TextStyle(fontSize: 20);
```

Comment:
This name is a bit unspecific — the whole page is the profile, so it doesn't help much in narrowing down which text this is for.

It seems like the answer is: currently name and role, and in the future perhaps email, user status, local time, etc. In other words the fields that are shown above the custom profile fields.

Perhaps `headingText`? Or `primaryFieldText`? Or could be `profileHeadingText` or `profilePrimaryFieldText`, though it's already private to the profile-page code so that doesn't seem essential.